modules/junit-jupiter/src/main/java/org/testcontainers/junit/jupiter/TestcontainersTestDescription.java

```diff
@@ -0,0 +1,20 @@
+package org.testcontainers.junit.jupiter;
+
+import lombok.RequiredArgsConstructor;
+import org.testcontainers.lifecycle.TestDescription;
+
+@RequiredArgsConstructor
+public class TestcontainersTestDescription implements TestDescription {
```

Comment:
```suggestion
class TestcontainersTestDescription implements TestDescription {
```

does not need to be public